samples/annotate/project-build.xml

```diff
@@ -159,10 +158,6 @@
 			<formatter type="xml"/>
 
 			<batchtest fork="yes" todir="${basedir}/target/junit-reports">
-				<fileset dir="${basedir}/src/test/java">
-					<include name="**/*Test*.java"/>
-					<exclude name="**/AllTests.java"/>
-				</fileset>
 				<fileset dir="${basedir}/target/generated-sources/xjc">
```

Comment:
Why this? src/main/java, etc. dirs are prepared in the project template so that users could easily add their code. I won't throw them out of the build.xml.